test/serialize.js

```diff
@@ -61,6 +131,14 @@ test('parse->serialize', function() {
       cookie.serialize('cat', ' ";/')));
 });
 
+test('encoded method validation', function() {
+   assert.throws(cookie.serialize.bind(null, 'cat', ' ', {
+       encode: 42
+   }), /number is not a function/);
```

Comment:
Why not just throw our own error, so the message is predictable (and probably more helpful to users)?